src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -437,120 +358,18 @@ paddingBottom
 paddingStart
 ```
 
-### Disabled
+### Component Disabled
 
 <Canvas>
   <Story name="Disabled">
     <div className={styles.boxWrapper}>
-      <Box disabled>Box.disabled.DISABLED</Box>
-    </div>
-  </Story>
-</Canvas>
-
-### Box composite example 1
-
-Panel Example with content and actionable footer
-
-<Canvas>
-  <Story name="Box composite example">
-    <div className={styles.boxWrapperExample}>
-      <Box shadow={Box.shadows.SMALL} rounded={Box.roundeds.MEDIUM}>
-        <Box
-          backgroundColor={Box.backgroundColors.GREY_BACKGROUND_COLOR}
-          padding={Box.paddings.MEDIUM}
-          borderBottom={Box.borderBottoms.DEFAULT}
-          borderColor={Box.borderColors.LAYOUT_BORDER_COLOR}
-        >
-          A composite panel header
-        </Box>
-        <Box paddingX={Box.paddingXs.MEDIUM} paddingY={Box.paddingYs.XL}>
-          <Flex direction={Flex.directions.COLUMN} justify={Flex.justify.CENTER}>
-            <Skeleton width={400} height={100} />
-            <Heading type={Heading.types.h3} value="Panel content heading" />
-            <p>Please approve the content of this paragraph</p>
-          </Flex>
-        </Box>
-        <Box
-          borderTop={Box.borderTops.DEFAULT}
-          borderColor={Box.borderColors.LAYOUT_BORDER_COLOR}
-          padding={Box.paddings.MEDIUM}
-        >
-          <Flex justify={Flex.justify.END} gap={Flex.gaps.SMALL}>
-            <Button kind={Button.kinds.TERTIARY}>Cancel</Button>
-            <Button>Ok</Button>
-          </Flex>
-        </Box>
+      <Box backgroundColor={Box.backgroundColors.ALL_GREY_BACKGROUND_COLOR} border={Box.borders.DEFAULT} disabled>
+        Box.disabled.DISABLED
       </Box>
     </div>
   </Story>
 </Canvas>
 
-### Box composite example 2
```

Comment:
in general we do like add use cases and examples from the real world (monday products) under use case and examples section. but we can do this in a different iteration under different task.

Reply:
@meiravron is working on that.
we has 2 examples but she wanted to rethink it.